velox/functions/lib/tests/QuantileDigestTest.cpp

```diff
@@ -952,4 +976,290 @@ TEST_F(QuantileDigestTest, getDistributionFunction) {
   testGetDistributionFunction<float>();
 }
 
+TEST_F(QuantileDigestTest, getValuesInQuantileRange) {
+  QuantileDigest<int64_t> digest{StlAllocator<int64_t>(allocator()), 0.01};
+
+  digest.add(1, 2.0);
+  digest.add(5, 3.0);
+  digest.add(10, 1.0);
+  digest.add(15, 4.0);
+
+  auto result1 = digest.getValuesInQuantileRange(0.1, 0.7);
+  EXPECT_GT(result1.size(), 0);
+
+  double totalWeight = 0.0;
+  for (const auto& pair : result1) {
+    totalWeight += pair.count;
+  }
+  EXPECT_NEAR(totalWeight, 6.0, 0.1);
```

Comment:
Would it make sense to just check the content in result1 directly? Something like
```
std::vector<decltype(digest)::ValueCountPair> expected = {{1, 1.0}, {5, 3.0}, {10, 1.0}, {15, 1.0}};
EXPECT_EQ(result1, expected);
```
Also, I think the algorithm you implemented is unlikely to cause imprecision issue of floating-point. If we can use EXPECT_EQ directly, no need to use EXPECT_NEAR.